repour/adjust/gradle_provider.py

```diff
@@ -15,7 +15,8 @@
 EXECUTION_NAME = "GRADLE"
 
 INIT_SCRIPT_FILE_NAME = "analyzer-init.gradle"
-MANIPULATION_FILE_NAME = "manipulation.json"
+MAIN_MANIPULATION_FILE_NAME = "alignmentReport.json"
+LEGACY_MANIPULATION_FILE_NAME = "manipulation.json"
```

Comment:
If I'm being pedantic, the manipulation result is still `manipulation.json` but there is a new alignment report (`alignmentReport.json`) to match PME.

Reply:
Is the format of manipulation.json the same as alignmentReport.json  for GME?

Reply:
No ; one is https://github.com/project-ncl/gradle-manipulator/blob/master/common/src/main/java/org/jboss/gm/common/model/ManipulationModel.java while the other is https://github.com/release-engineering/pom-manipulation-ext/blob/master/common/src/main/java/org/commonjava/maven/ext/common/json/PME.java

Reply:
Gotcha! Thanks for pointing this out. That means this PR is wrong and I'll adjust it accordingly.